package.json

```diff
@@ -158,7 +159,9 @@
     "wpt-all": {
       "command": "tools/runWPTAll.sh",
       "files": [
-        "tools/runWPTAll.sh"
+        "tools/run-wpt.mjs",
+        "tools/runWPTAll.sh",
+        "wpt/**/*.py"
```

Comment:
Scope it lower to `wpt/webdriver/tests/**` that way it doesn't take thousand of files to watch.

Reply:
It's not comprehensive enough.

For example: We also care about `tools/webdriver/webdriver/**/*.py`.

I could change to match these two entries at least, but there could be others I'm unaware of.

Reply:
Done.